ray-operator/controllers/ray/raycluster_controller.go

```
@@ -121,27 +123,41 @@ func (r *RayClusterReconciler) eventReconcile(request ctrl.Request, event *corev
 	// we only care about pod events
 	if event.InvolvedObject.Kind != "Pod" || event.Type != "Warning" || event.Reason != "Unhealthy" ||
 		!strings.Contains(event.Message, "Readiness probe failed") {
+		// This is not supposed to happen
```

Should we surface the issue here rather than `return` without doing anything?